user-manual/administer/settings.rst

```diff
@@ -1871,3 +1872,95 @@ setting.
 
 
 :ref:`Back to top <settings>`
+
+.. _privacy-notification:
+
+Privacy notification
+====================
+
+.. NOTE::
+   This feature is new in the AtoM 2.4.1 release and is not present or available
+   in any previous versions (see `Release Notes 2.4.1 <https://wiki.accesstomemory.org/Releases/Release_announcements/Release_2.4.1>`)
+
+We have added a new default static page called "Privacy Policy" (available under the Quick links menu in AtoM header), and a new configurable cookie notification banner in the AtoM 2.4.1 release to comply with the European Union's General Data Protection Regulation (GDPR). The GDPR came into effect on May 1, 2018, and even if your institution is not situtated in the EU, it is
+important to be aware of this regulation and how it might affect you.
+
+In compliance with the GDPR, AtoM now makes explicit its collection and use of cookies. AtoM collects cookies in order to enable browsing and loading of certain types of content. Visitors to AtoM sites who do not wish to have cookies placed on their computers should set their browsers to refuse cookies. However, certain features may not function properly without the aid of cookies.
+
+AtoM supports integration with `Google Analytics <https://www.google.com/analytics/>` for the purposes of gathering statistics on page views, site usage, user location, and other data on site visits. All data collected by Google Analytics are stored and processed by Google, according to the `Google Ads Data Processing Terms <https://privacy.google.com/businesses/processorterms/>`.
```

Comment:
Could potentially add a link to this section of the Admin manual: https://www.accesstomemory.org/docs/latest/admin-manual/maintenance/web-analytics/#page-tagging-tools